dependencies.gradle

```diff
@@ -25,6 +25,7 @@ depVersions = [
     arquillianCubeDocker: "1.18.2",
     arquillianJunit: "1.6.0.Final",
     bcFips: "1.0.1",
+    codehale: "3.1.0",
```

Comment:
the old name was "codahale" with "A", but it was anyway renamed to dropwizard

Reply:
Its just a variable I will rename it to dropwizard.